### ISSUE TITLE
W098 warning missing

### ISSUE THREAD

**Issue by peonmodel** [AtomLinter/linter-jshint#338]
There is a bug in detecting W098 warning, the hinter only checks for the last param declared in function is used and ignores the rest

```javascript
function test(a,b,c){
  console.log(a);  // this gives a W098 'c'&'b' is defined but never used warning
}
test();
```
```javascript
function test(a,b,c){
  console.log(a, c);  // this DOES NOT gives a W098 'b' is defined but never used warning
}
test();
```

**Comment by Arcanemagus** [comment]
This looks like a bug in JSHint, are you able to reproduce it on the CLI?

**Comment by Arcanemagus** [comment]
Looks like https://github.com/jshint/jshint/issues/1994 is the issue you want to follow (and possibly bump).

**Comment by Arcanemagus** [comment]
Just as an FYI, and to help if you think this needs to be reported as a new issue over there, I tried to reproduce with the following files: [linter-jshint_GH338.zip](https://github.com/AtomLinter/linter-jshint/files/480667/linter-jshint_GH338.zip)

The CLI interface matches your described behavior exactly, so this has nothing to do with this package.
